example-app/src/main/res/layout/activity_main.xml

```diff
@@ -46,7 +46,7 @@
         android:id="@+id/dumpWithoutCrash"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="DumpWithoutCrash"
+        android:text="Generate sample crash report"
```

Comment:
why?

Reply:
DumpWIthoutCrash is our internal interface name. It doesn't tall the user what it does. "Generate sample crash report" tells exactly what user can expect by pressing the button